lifecycle-operator/apis/lifecycle/v1alpha4/keptnworkloadversion_types.go

```diff
@@ -83,19 +83,6 @@ type KeptnWorkloadVersionStatus struct {
 	Status common.KeptnState `json:"status,omitempty"`
 }
 
-type ItemStatus struct {
```

Comment:
If we use `v1alpha4.ItemStatus` in `KeptnWorkloadVersion`, it will not implement the `PhaseItem` interface which will lead to the need of changing the interface -> this will cause that `KeptnAppVersion` will not implement the updated interface anymore and therefore will need to be added to the `v1alpha4` API version. This will cause a domino effect and probably we will end up just adding all of the CRDs to the v1alpha4, which we do not want. Therefore it is easier to use `v1alpha3.ItemStatus` in `KeptnWorkloadVersion` and then we can remove the `ItemStatus` from `v1alpha4` as it is not used anywhere.

Hopefully it's a little bit clearer :)

Reply:
That was a perfect explanation :) Thanks a lot 🙌